server/models/db/workerTimeslot.js

```diff
@@ -21,7 +21,19 @@ async function updateWorkerAvailability(workerId, specificDate, newStatus) {
     return db.any(`update ${Tables.workerTimeslot} set status='${newStatus}' where date='${specificDate}' and worker_id='${workerId}';`);
 }
 
+async function checkWorkerAvailability(workerTimeslotId) {
+    //Selects all from workerTimeslot table 
+    const queryStatement = `select * from ${Tables.workerTimeslot}  where worker_timeslot_id = ${workerTimeslotId}`;
```

Comment:
Hi Nancy,
I would suggest wrapping lines 25 to 32 in a try and catch block similar to the following to account for potential problems in the database query. Remember to right click and format the code too to increase code readability.
![image](https://user-images.githubusercontent.com/32720322/100938022-eded0c80-34c1-11eb-993a-5debe87b72f1.png)

Reply:
Hi Amy,
I'm facing the following issues when implementing your suggestion, do you have any suggestions:
<img width="571" alt="Screen Shot 2020-12-02 at 5 20 35 PM" src="https://user-images.githubusercontent.com/71143782/100938502-c8143780-34c2-11eb-9c53-6605ae0532ed.png">

Reply:
Hi Amy, 
Thank you for connecting with me offline; I have resolved the aforementioned issue and pushed the changes.  
You can see this change in lines 25-36 of server/models/db/workerTimeslot.js

Please confirm if this takes care of your suggestion!